.circleci/config.yml

```diff
@@ -210,6 +220,67 @@ jobs:
                   [ -n "<< pipeline.parameters.PACKAGE_INSTALL_COMMAND >>" ] && eval "<< pipeline.parameters.PACKAGE_INSTALL_COMMAND >>"
                   eval "<< pipeline.parameters.BUILD_COMMAND >>"
 
+      - when:
+          condition:
+            equal: ["nextjs", "<< pipeline.parameters.FRAMEWORK_TYPE >>"]
+          steps:
+            - run:
+                name: Build Next.js Project
+                working_directory: ~/project/repository
+                command: |
+                  set -e
+
+                  # Function to log messages
+                  log() {
+                    echo "$(date +'%Y-%m-%d %H:%M:%S') - $1"
+                  }
+
+                  # Load environment variables
+                  log "Loading environment variables..."
+                  [ -f ~/project/.env ] && source ~/project/.env
+                  [ -f ~/project/.env.ef_circleci ] && source ~/project/.env.ef_circleci
+
+                  # Change to the specified build directory if provided
+                  if [ -n "<< pipeline.parameters.BUILD_DIR >>" ]; then
+                    log "Changing to build directory: << pipeline.parameters.BUILD_DIR >>"
+                    cd "<< pipeline.parameters.BUILD_DIR >>"
+                  fi
+
+                  # Install dependencies
+                  if [ -n "<< pipeline.parameters.PACKAGE_INSTALL_COMMAND >>" ]; then
+                    log "Installing dependencies..."
+                    eval "NEXT_PRIVATE_STANDALONE=true << pipeline.parameters.PACKAGE_INSTALL_COMMAND >>"
+                  else
+                    log "Error: PACKAGE_INSTALL_COMMAND is not set. Exiting."
+                    exit 1
+                  fi
+
+                  # Build the project
+                  if [ -n "<< pipeline.parameters.BUILD_COMMAND >>" ]; then
+                    log "Building the project..."
+                    eval "NEXT_PRIVATE_STANDALONE=true << pipeline.parameters.BUILD_COMMAND >>"
+                  else
+                    log "Error: BUILD_COMMAND is not set. Exiting."
+                    exit 1
+                  fi
+
+                  # Create output directory
+                  log "Creating output directories..."
+                  mkdir -p output output/public output/.next output/node_modules
+
+                  # Move necessary files from standalone to output
+                  log "Moving files to output directory..."
+                  mv .next/standalone/* output/
+                  mv .next/standalone/.next output/
```

Comment:
i think this command might be unnecessary because the previous like wildcard should copy everything in standalone including hidden folders to output

Reply:
I tested it and it didn't copy the .next folder inside standalone folder, for that reason I added the next line `mv .next/standalone/.next output/`

Reply:
Oh you're using mv instead of "cp -r" that might be why. Okay that's probably fine though